chainermn/communicators/mpi_communicator_base.py

```diff
@@ -322,16 +338,40 @@ def gather(self, x, root=0):
         else:
             return None
 
-    def broadcast_data(self, model):
-        raise NotImplementedError()
+    # Objects
+    def send_obj(self, obj, dest):
+        self.mpi_comm.send(obj, dest=dest)
+
+    def recv_obj(self, source):
+        return self.mpi_comm.recv(source=source)
+
+    def bcast_obj(self, obj, max_buf_len=None, root=0):
+        if max_buf_len is None:
+            max_buf_len = 256 * 1024 * 1024
```

Comment:
why don't we just set the default value of `max_buf_len` to `256*1024*1024` instead of `None` ?